query/physicalplan/physicalplan.go

```diff
@@ -277,6 +277,12 @@ func WithOrderedAggregations() Option {
 	}
 }
 
+func WithConcurrency(concurrency int) Option {
```

Comment:
Can you add a comment here? We also need to mention default value.

Reply:
Good call!